static/js/formatters-internal.js

```diff
@@ -264,6 +264,7 @@ export function prettyPrintObject(obj, locale) {
  * @returns {string} The localized affirmative or negative.
  */
 function _prettyPrintBoolean(value, locale) {
+  // TODO: Use utility method for getting lanugage as this logic doesn't work for Chinese.
```

Comment:
I merged in the parseLocale method so if you want to rebase you could use that here